stl/inc/xstring

```diff
@@ -2536,14 +2555,14 @@ public:
         _Construct<_Construct_strategy::_From_ptr>(_Ptr, _Count);
     }
 
-    _CONSTEXPR20 basic_string(_In_z_ const _Elem* const _Ptr) : _Mypair(_Zero_then_variadic_args_t{}) {
-        _Construct<_Construct_strategy::_From_ptr>(_Ptr, _Convert_size<size_type>(_Traits::length(_Ptr)));
-    }
-
 #if _HAS_CXX23
     basic_string(nullptr_t) = delete;
 #endif // _HAS_CXX23

```

Comment:
This is a desirable change to avoid splitting the constructor implementation, but `basic_string(nullptr_t)` is ending up in the "wrong" place compared to its appearance in the Standard. We're far from consistent (*especially* in old code), but nowadays we try to match the Standard's declaration order unless there's a good reason otherwise.